rules/windows/defense_evasion_wsl_bash_exec.toml

```diff
@@ -22,23 +22,31 @@ references = [
     "https://lolbas-project.github.io/lolbas/OtherMSBinaries/Wsl/",
     "https://blog.qualys.com/vulnerabilities-threat-research/2022/03/22/implications-of-windows-subsystem-for-linux-for-adversaries-defenders-part-1"
 ]
-risk_score = 47
+risk_score = 21
 rule_id = "3e0eeb75-16e8-4f2f-9826-62461ca128b7"
-severity = "medium"
+severity = "low"
 tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Execution", "Tactic: Defense Evasion", "Data Source: Elastic Endgame", "Data Source: Elastic Defend"]
 timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type : "start" and
- (
-  ((process.executable : "?:\\Windows\\System32\\bash.exe" or process.pe.original_file_name == "Bash.exe") and 
-  not process.command_line : ("bash", "bash.exe")) or 
-  process.executable : "?:\\Users\\*\\AppData\\Local\\Packages\\*\\rootfs\\usr\\bin\\bash" or 
-  (process.parent.name : "wsl.exe" and process.parent.command_line : "bash*" and not process.name : "wslhost.exe") or 
-  (process.name : "wsl.exe" and process.args : ("curl", "/etc/shadow", "/etc/passwd", "cat","--system", "root", "-e", "--exec", "bash", "/mnt/c/*"))
+  (
+    (
+      (process.executable : "?:\\Windows\\System32\\bash.exe" or process.pe.original_file_name == "Bash.exe") and 
+      not process.command_line : ("bash", "bash.exe")
+    ) or 
+    process.executable : "?:\\Users\\*\\AppData\\Local\\Packages\\*\\rootfs\\usr\\bin\\bash" or 
+    (
+      process.parent.name : "wsl.exe" and process.parent.command_line : "bash*" and not process.name : "wslhost.exe"
+    ) or 
+    (
+      process.name : "wsl.exe" and process.args : (
+        "curl", "/etc/shadow", "/etc/passwd", "cat", "--system", "root", "-e", "--exec", "bash", "/mnt/c/*"
+      ) and not process.args : ("wsl-bootstrap", "docker-desktop-data", "*.vscode-server*")
```

Comment:
Actual changes to logic, the rest is to make it more readable

Reply:
Does it make sense to flag on `process.args` of `cat`, `root`, `-e`? To me that sounds a bit generic. I would suggest adding:

- [Tunneling related activity](https://github.com/elastic/detection-rules/blob/main/rules/linux/command_and_control_linux_tunneling_and_port_forwarding.toml)
- [Specific tools such as nc, or another utility that could potentially spawn reverse connections](https://github.com/elastic/detection-rules/blob/main/rules/linux/execution_shell_via_lolbin_interpreter_linux.toml)
- [File alterations in specific suspicious directories](https://github.com/elastic/detection-rules/blob/main/rules/linux/persistence_etc_file_creation.toml)
- Tools that can help initialize network connections such as `wget`. 

Might be a large change, LMK if you would like to add some of these changes. Can sync via Slack.

Reply:
These additions will be added in seperate rules, and are tracked in https://github.com/elastic/detection-rules/issues/3362.